### PR TITLE
LPS-73719

### PR DESCRIPTION
**Pull request by jeyvison**

### CODE REVIEW

**Comment by jeyvison** [comment]
ci:retest

**Comment by vicnate5** [comment]
Test failures not related.

File: portal-workflow-web/src/main/resources/META-INF/resources/js/main.js [hunk 1]
```diff
@@ -2,6 +2,62 @@ AUI.add(
 	'liferay-workflow-web',
 	function(A) {
 		var WorkflowWeb = {
+			previewBeforeRevert: function(event, renderUrl, actionUrl, title) {
+
```
Comment (inacionery) [inline portal-workflow-web/src/main/resources/META-INF/resources/js/main.js]: Hey @jeyvison, please add `var instance = this;` at the beginning of the function is a standard. thx!

File: portal-workflow-web/src/main/resources/META-INF/resources/js/main.js [hunk 2]
```diff
@@ -34,13 +90,13 @@ AUI.add(
 
 				instance._alert = alert;
 			},
-
```
Comment (inacionery) [inline portal-workflow-web/src/main/resources/META-INF/resources/js/main.js]: Hey @jeyvison, please keep the line. thx!

File: portal-workflow-web/src/main/resources/META-INF/resources/definition/view_workflow_definition.jsp [hunk 3]
```diff
@@ -62,7 +69,11 @@ renderResponse.setTitle(workflowDefinition.getName());
 						<%= dateFormatTime.format(workflowDefinition.getModifiedDate()) %>
 					</c:when>
 					<c:otherwise>
+						<% if(WorkflowWebKeys.WORKFLOW_PREVIEW_BEFORE_RESTORE_STATE.equals(state)) { %>
```
[the remark this answers is not in the document]
Comment (jeyvison) [reply portal-workflow-web/src/main/resources/META-INF/resources/definition/view_workflow_definition.jsp]: Hey @inacionery , i understand it is a default choice to use c:choose when we are evaluating something in JSP but in this case a c:choose nested to another c:choose would let it even more verbose then it already is. Change it anyway?

[the remark this answers is not in the document]
Comment (inacionery) [reply portal-workflow-web/src/main/resources/META-INF/resources/definition/view_workflow_definition.jsp]: Could you use `c:when` instead?

[the remark this answers is not in the document]
Comment (jeyvison) [reply portal-workflow-web/src/main/resources/META-INF/resources/definition/view_workflow_definition.jsp]: I'll give it a try :)

**Comment by inacionery** [comment]
Hey @jeyvison, this PR has conflict, could you please resolve it. Thx! We als have a project that verifies the SF for JavaScript, HTML and CSS files https://github.com/natecavanaugh/check-source-formatting, it's always good run it. 😉

**Comment by jeyvison** [comment]
@inacionery Changes made :)

**Comment by inacionery** [comment]
Hey @brandizzi, could you please review this PR. Thx!

**Comment by inacionery** [comment]
ci:retest

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jeyvison** [comment]
@brandizzi Changes made

**Comment by jeyvison** [comment]
ci:retest

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by inacionery** [comment]
ci:retest

**Comment by inacionery** [comment]
ci:stop

**Comment by inacionery** [comment]
ci:retest

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by brandizzi** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by inacionery** [comment]
Forwarded to https://github.com/liferay/com-liferay-portal-workflow/pull/252

**Pull request closed without merging**